Makefile

```diff
@@ -122,7 +126,7 @@ endif
 clean: clean-dunst clean-dunstify clean-doc clean-tests clean-coverage clean-coverage-run
 
 clean-dunst:
-	rm -f dunst ${OBJ} main.o
+	rm -f dunst ${OBJ} main.o main.d ${DEPS}
```

Comment:
:thinking: If we have to manually remove `main.d` here (because it's not in the `DEPS` variable, we don't include it in the build process.

Reply:
What about adding `main.c` into `SRC`? This might fix all extra mentions for `main.*?

Reply:
> thinking If we have to manually remove main.d here (because it's not in the DEPS variable, we don't include it in the build process.

That's exactly what `main.c` is there for. If we include it in `SRC` it'll get linked along with the tests and it will fail for having 2 mains.